swagger_parser/swagger_parser.py

```diff
@@ -355,7 +357,15 @@ def get_paths_data(self):
         for path, path_spec in self.specification['paths'].items():
             self.paths[path] = {}
 
+            # Add path-level parameters
+            default_parameters = {}
+            if 'parameters' in path_spec:
+                self._add_parameters(default_parameters, path_spec['parameters'])
+
             for action in path_spec.keys():
+                if action not in self._HTTP_VERBS:
```

Comment:
This seemed an appropriate check given that the logic below this line assumes `path_spec[action]` is an [Operation Object](http://swagger.io/specification/#operationObject), but that only appears to be true if `action` is one of these HTTP verbs. (Other possible values include `$ref` and `parameters`.)